.expeditor/update_chef.sh

```diff
@@ -13,17 +13,24 @@ set -evx
 
 branch="expeditor/chef_${VERSION}"
 git checkout -b "$branch"
+GEM_NAME="chef"
 
 # make sure we have rake for the tasks later
-bundle install
-sed -i -r "s/^\s*gem \"chef\".*/  gem \"chef\", \"= ${VERSION}\"/" Gemfile
-
-# it appears that the gem that triggers this script fires off this script before
-# the gem is actually available via bundler on rubygems.org.
-sleep 240
-
-gem install rake
-rake dependencies:update_gemfile_lock
+gem install bundler -v 1.17.3 --no-document
```

Comment:
We should grab the version here out of our omnibus_overrides file like we do in travis. That way this doesn't break when we bump the version there.